publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -416,15 +427,17 @@ private boolean hasExchange() throws RemRemPublishException {
         try {
             channel = connection.createChannel();
         } catch (final IOException e) {
-            log.info("Exchange " + exchangeName + " does not Exist");
+            log.info("Exchange " + exchangeName + " does not exist");
```

Comment:
" does not exist" - well, yes, it does not exist because it was not created. is not something like this better suited:

" was not created" ?